docs/how-to/ccv0.sh

```diff
@@ -51,6 +51,10 @@ export CONTAINER_ID="${CONTAINER_ID:-0123456789}"
 source /etc/os-release || source /usr/lib/os-release
 grep -Eq "\<fedora\>" /etc/os-release 2> /dev/null && export USE_PODMAN=true
 
+if [ -d ${tests_repo_dir} ]; then
+    export BATS_TEST_DIRNAME="${tests_repo_dir}/integration/containerd/confidential/"
```

Comment:
The name `BATS_TESTS_DIRNAME` is a special variable of bats (see [here](https://bats-core.readthedocs.io/en/stable/writing-tests.html#special-variables)). It might silently introduce a bug If in the future `ccv0.sh` calls bats due to clash on the variable names.